fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -2535,4 +2537,37 @@ public List<Tablet> getAllTablets() throws AnalysisException {
         }
         return tablets;
     }
+
+    @Override
+    public PartitionType getPartitionType() {
+        return partitionInfo.getType();
+    }
+
+    @Override
+    public Map<Long, PartitionItem> getPartitionItems() {
+        return getPartitionInfo().getIdToItem(false);
+    }
+
+    @Override
+    public long getPartitionLastModifyTime(long partitionId, PartitionItem item) throws AnalysisException {
+        return getPartitionOrAnalysisException(partitionId).getVisibleVersionTimeIgnoreInit();
+    }
+
+    @Override
+    public long getLastModifyTime() {
```

Comment:
what will the last modify time be if a partition is dropped?

Reply:
olap table has same problem,resolve together with other PRs